sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -15,13 +15,17 @@ public partial class BackupOperation : Azure.Operation<Azure.Security.KeyVault.A
         public override System.Threading.Tasks.ValueTask<Azure.Response<Azure.Security.KeyVault.Administration.BackupResult>> WaitForCompletionAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.ValueTask<Azure.Response<Azure.Security.KeyVault.Administration.BackupResult>> WaitForCompletionAsync(System.TimeSpan pollingInterval, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
-    public partial class BackupResult
+    public abstract partial class BackupRestoreResult
     {
-        internal BackupResult() { }
-        public System.Uri BackupFolderUri { get { throw null; } }
+        internal BackupRestoreResult() { }
         public System.DateTimeOffset EndTime { get { throw null; } }
         public System.DateTimeOffset StartTime { get { throw null; } }
     }
+    public partial class BackupResult : Azure.Security.KeyVault.Administration.BackupRestoreResult
+    {
+        internal BackupResult() { }
+        public System.Uri folderUri { get { throw null; } }
```

Comment:
Properties should be PascalCase. FYI: search/replace has a mode to keep casing in VS; I find refactoring tool / rename works great in cases like this.